runtests.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
-PYTHONPATH=anitya ./nosetests \
```

Comment:
No, as the current directory gets added to sys.path implicitly, and that's all that's needed to make "anitya" importable. Actually adding the package directory to the path is potentially problematic, as it sets up the ability to import the same component twice under different names: http://python-notes.curiousefficiency.org/en/latest/python_concepts/import_traps.html#the-double-import-trap